macos/Sources/Features/QuickTerminal/QuickTerminalWindow.swift

```diff
@@ -28,23 +28,5 @@ class QuickTerminalWindow: NSPanel {
 
         // We don't want to activate the owning app when quick terminal is triggered.
         self.styleMask.insert(.nonactivatingPanel)
-
-        // We need to set our window level to a high value. In testing, only
-        // popUpMenu and above do what we want. This gets it above the menu bar
-        // and lets us render off screen.
-        self.level = .popUpMenu
-
-        // This plus the level above was what was needed for the animation to work,
-        // because it gets the window off screen properly. Plus we add some fields
-        // we just want the behavior of.
-        self.collectionBehavior = [
-            // We want this to be part of every space because it is a singleton.
-            .canJoinAllSpaces,
-
-            // We don't want to be part of command-tilde
-            .ignoresCycle,
-
-            // We want to show the window on another space if it is visible
-            .fullScreenAuxiliary]
```

Comment:
All of this now gets managed by the controller so we don't need to set it on the window.